backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/UpdateDiskCommand.java

```diff
@@ -786,7 +788,12 @@ public String getDiskAlias() {
     public Map<String, String> getJobMessageProperties() {
         if (jobProperties == null) {
             jobProperties = super.getJobMessageProperties();
-            jobProperties.put("diskalias", getDiskAlias());
```

Comment:
we'll then miss the disk alias, no?

Reply:
Not really, it is already added in `super.getJobMessageProperties()`, that is in `AbstractDiskVmCommand` so IMHO just a redundant code, unless I miss something.
Obviously I also tested it manually (with a debugger) and saw that `diskalias` is there.

Reply:
ack